frontend/src/interfaces/accessRequests.tsx

```diff
@@ -0,0 +1,39 @@
+export interface IUserInfo {
```

Comment:
I was finding it rather random where interfaces were being stored for data models.  Maybe it was just my lack of understanding, but I thought it might be easier if they were in their own location and perhaps had a similar structure as those in the backend.  Please let me know your thoughts on this?

Reply:
That works for me, although I don't think I agree with userInfo being part of AccessRequests.

Reply:
moved